swift/vapor/main.swift

```diff
@@ -1,20 +0,0 @@
-import Vapor
-
-var services = Services.default()
-services.register(MiddlewareConfig())
-
-let router = EngineRouter.default()
-let empty = HTTPResponse()
```

Comment:
Allocating the response outside of the request handlers should be more performant.

Reply:
True, but then aren't we just gaming the system?

Reply:
That's up to the benchmark maintainers to decide I think. The purpose of this PR is to update from v3 to v4 and this change goes beyond that.